systers_portal/meetup/forms.py

```diff
@@ -104,14 +112,25 @@ def clean_time(self):
 class EditMeetupForm(ModelFormWithHelper):
     """Form to edit Meetup"""
 
+    images = MultiFileField(required=False)
+
     class Meta:
         model = Meetup
-        fields = ('title', 'slug', 'date', 'time', 'description', 'venue')
+        fields = ('title', 'slug', 'date', 'time', 'description', 'venue',)
```

Comment:
Why have you added an extra comma?

Reply:
I was trying out different ways so left the comma there. Will remove it :)